tools/clang/test/HLSLFileCheck/d3dreflect/bitfield_spill-over.hlsl

```diff
@@ -0,0 +1,141 @@
+// CHECK-NEXT:                       Type: D3D_SVT_INT
+// CHECK-NEXT:                       Elements: 0
+// CHECK-NEXT:                       Rows: 1
+// CHECK-NEXT:                       Columns: 8
+// CHECK-NEXT:                       Members: 0
+// CHECK-NEXT:                       Offset: 0
+// CHECK-NEXT:                   ID3D12ShaderReflectionType:
+// CHECK-NEXT:                     D3D12_SHADER_TYPE_DESC: Name: int
+// CHECK-NEXT:                       Class: D3D_SVC_BIT_FIELD
+// CHECK-NEXT:                       Type: D3D_SVT_INT
+// CHECK-NEXT:                       Elements: 0
+// CHECK-NEXT:                       Rows: 1
+// CHECK-NEXT:                       Columns: 8
+// CHECK-NEXT:                       Members: 0
+// CHECK-NEXT:                       Offset: 8
+// CHECK-NEXT:                   ID3D12ShaderReflectionType:
+// CHECK-NEXT:                     D3D12_SHADER_TYPE_DESC: Name: int
+// CHECK-NEXT:                       Class: D3D_SVC_BIT_FIELD
+// CHECK-NEXT:                       Type: D3D_SVT_INT
+// CHECK-NEXT:                       Elements: 0
+// CHECK-NEXT:                       Rows: 1
+// CHECK-NEXT:                       Columns: 8
+// CHECK-NEXT:                       Members: 0
+// CHECK-NEXT:                       Offset: 16
+// CHECK-NEXT:                   ID3D12ShaderReflectionType:
+// CHECK-NEXT:                     D3D12_SHADER_TYPE_DESC: Name: int
+// CHECK-NEXT:                       Class: D3D_SVC_BIT_FIELD
+// CHECK-NEXT:                       Type: D3D_SVT_INT
+// CHECK-NEXT:                       Elements: 0
+// CHECK-NEXT:                       Rows: 1
+// CHECK-NEXT:                       Columns: 8
+// CHECK-NEXT:                       Members: 0
+// CHECK-NEXT:                       Offset: 24
+// CHECK-NEXT:                 }
+// CHECK-NEXT:               ID3D12ShaderReflectionType:
+// CHECK-NEXT:                 D3D12_SHADER_TYPE_DESC: Name: int
+// CHECK-NEXT:                   Class: D3D_SVC_SCALAR
+// CHECK-NEXT:                   Type: D3D_SVT_INT
+// CHECK-NEXT:                   Elements: 0
+// CHECK-NEXT:                   Rows: 1
+// CHECK-NEXT:                   Columns: 1
+// CHECK-NEXT:                   Members: 3
+// CHECK-NEXT:                   Offset: 4
+// CHECK-NEXT:                 {
+// CHECK-NEXT:                   ID3D12ShaderReflectionType:
+// CHECK-NEXT:                     D3D12_SHADER_TYPE_DESC: Name: int
+// CHECK-NEXT:                       Class: D3D_SVC_BIT_FIELD
+// CHECK-NEXT:                       Type: D3D_SVT_INT
+// CHECK-NEXT:                       Elements: 0
+// CHECK-NEXT:                       Rows: 1
+// CHECK-NEXT:                       Columns: 8
+// CHECK-NEXT:                       Members: 0
+// CHECK-NEXT:                       Offset: 0
+// CHECK-NEXT:                   ID3D12ShaderReflectionType:
+// CHECK-NEXT:                     D3D12_SHADER_TYPE_DESC: Name: int
+// CHECK-NEXT:                       Class: D3D_SVC_BIT_FIELD
```

Comment:
I'm torn.  Should padding be reflected here?  I don't suppose it hurts, but we also have offsets for all the named fields, so we don't technically need it, right?  When we add field names, I guess the name of this field would be nullptr, or "\0"?

Reply:
Add padding filed seems redundant like you mentioned.
For the name, I'm not sure. Both nullptr and "\0" feels OK.